docs/en/ingest-management/tab-widgets/add-fleet-server/content.asciidoc

```diff
@@ -52,20 +57,65 @@ For more information about these settings, see <<fleet-settings>>.
 [role="screenshot"]
 image::images/add-fleet-server.png[In-product instructions for adding a {fleet-server}]
 
+//QUESTION: Why does the Fleet Server install command include the --url flag
+//when Production mode is selected)? It's not there in Quick Start mode.
+//I thought --url was only required when you are enrolling subsequent agents in
+// Fleet Server.
```

Comment:
I'm not sure the url is needed here. Can you confirm @nimarezainia @blakerouse?

Reply:
@mostlyjason Ah so it looks like Pius opened an issue in the beats repo saying that --url is required when certs are specified: https://github.com/elastic/beats/issues/27245

Reply:
...but I do remember testing those commands fairly rigorously, so I'm pretty sure the --url setting was not always required when passing certs. Would be good to get clarification from the dev team about this.